src/ansys/mapdl/core/logging.py

```diff
@@ -108,13 +108,34 @@
 these loggers.
 
 """
-
 from copy import copy
 from datetime import datetime
 import logging
 import sys
+from types import TracebackType
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Dict,
+    Literal,
+    Mapping,
+    MutableMapping,
+    Optional,
+    Type,
+    Union,
+    cast,
+)
 import weakref
 
+if TYPE_CHECKING:  # pragma: no cover
+    from ansys.mapdl.core.mapdl import _MapdlCore
+
+    _LoggerAdapter = Type[logging.LoggerAdapter[logging.Logger]]
+    _StreamHandler = Type[logging.StreamHandler[Any]]
```

Comment:
We know that the problem is from here because using `logging.LoggerAdapter` instead of `logging.LoggerAdapter[logging.Logger]` makes it work

Reply:
The warning happen on the lines where these type are used

Reply:
Interesting. I do not have much experience typing. 


However  if we are using a `LoggerAdapter` then why the `[logging.Logger]`? I would probably just do:
```
_LoggerAdapter = Type[logging.LoggerAdapter]
```

Reply:
But one thing that I don't understand is why I don't get this error when I am building locally

Reply:
> Interesting. I do not have much experience typing.
> 
> However if we are using a `LoggerAdapter` then why the `[logging.Logger]`? I would probably just do:
> 
> ```
> _LoggerAdapter = Type[logging.LoggerAdapter]
> ```
> 
> Is this possible/reasonable?

That's what they did in pygeometry
it's not the best but it's still a usable

Reply:
You really dug deep into this. I am afraid I cannot give you any advice. Do whatever you think it is more appropriate.

Reply:
For now this will be fine I think